### PR TITLE
Google speech commands wrapper.

### PR DESCRIPTION
**Pull request by ebezzam**
Adding wrapper for the Google Speech Commands Dataset (https://research.googleblog.com/2017/08/launching-speech-commands-dataset.html).

There is also a example usage script in the "examples" folder.

### CODE REVIEW

File: pyroomacoustics/datasets/google_speech_commands.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    import sounddevice as sd
+    have_sounddevice = True
+except:
+    have_sounddevice = False
+
+from .utils import download_uncompress
+from .base import Meta, AudioSample, Dataset
+
+
+google_speech_commands_sounds = {
```
Comment (fakufaku) [inline pyroomacoustics/datasets/google_speech_commands.py]: I guess here `True/False` might be more explicit than `1/0`.

File: pyroomacoustics/datasets/google_speech_commands.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    'go' : {'speech' : 1},
+    'four' : {'speech' : 1},
+    'five' : {'speech' : 1},
+    'eight' : {'speech' : 1},
+    'down' : {'speech' : 1},
+    'dog' : {'speech' : 1},
+    'cat' : {'speech' : 1},
+    'bird' : {'speech' : 1},
+    'bed' : {'speech' : 1},
+    '_background_noise_' : {'speech' : 0},
```
Comment (fakufaku) [inline pyroomacoustics/datasets/google_speech_commands.py]: Is there a special reason to have the `_` (underscore) here ?

File: pyroomacoustics/datasets/google_speech_commands.py (hunk 1 above)
Comment (ebezzam) [reply pyroomacoustics/datasets/google_speech_commands.py]: `1` or I mean `True` ;)

I'll change to this.

Comment (fakufaku) [reply pyroomacoustics/datasets/google_speech_commands.py]: Ha ha :)

File: pyroomacoustics/datasets/google_speech_commands.py (hunk 2 above)
Comment (fakufaku) [reply pyroomacoustics/datasets/google_speech_commands.py]: No worries, that sounds like a good reason! I was just curious.

**Comment by ebezzam** [comment]
Cool, I'm going to have to create a new branch after the squash (that was my problem before). So I'll close this PR and open a new one then merge.

**Pull request closed without merging**